src/choreo/vote/fd_vote.c

```diff
@@ -1,34 +1,50 @@
 #include "fd_vote.h"
+#include <string.h>
 
 #pragma GCC diagnostic ignored "-Wformat"
 #pragma GCC diagnostic ignored "-Wformat-extra-args"
 
 ulong
 fd_vote_txn_generate(fd_compact_vote_state_update_t* compact_vote_update,
-                     fd_pubkey_t* validator_pubkey,
-                     fd_pubkey_t* vote_acct_pubkey,
-                     uchar* validator_privkey,
-                     uchar* vote_acct_privkey,
+                     fd_pubkey_t* node_pubkey,
+                     fd_pubkey_t* authorized_voter_pubkey,
+                     fd_pubkey_t* vote_acct_addr,
+                     uchar* node_privkey,
+                     uchar* authorized_voter_privkey,
                      uchar* recent_blockhash,
```

Comment:
formatting *

Reply:
I just noticed this and I have revised all files in this PR for this pointer formatting.